doc/user_guide/user_guide.md

```diff
@@ -466,6 +466,138 @@ CREATE OR REPLACE JAVA SET SCRIPT IMPORT_FILES(...) EMITS (...) AS
+resources (number of cores, memory per node), the number of files and the size
+of each file.
+
+### Export Parallelism Parameter
+
+In the export, the parallelism works differently compared to the import SQL
+statement.
+
+In the import statement, we are importing data from many files. Using the user
+provider parallelism number, we distribute these files into that many importer
+processes. For example, simply by taking modulo of file hash by parallelism
+number.
+
+In export, we have a table with many records. When exporting an Exasol table,
+the `PARALLELISM` parameter value is internally used in a `GROUP BY` clause to
+distribute the table records into many exporter processes. The parallelism
+should be something dynamic that Exasol database can understand and use in the
+group by clause.
+
+The default value for parallelism for export is `iproc()` (notice that it is
+different from `nproc()`). It returns the data node id numbers. Therefore,
+by default, it creates as many exporter processes as the number of datanodes.
+
+```sql
+EXPORT <schema>.<table>
+INTO SCRIPT CLOUD_STORAGE_EXTENSION.EXPORT_PATH WITH
+  BUCKET_PATH     = 's3a://<S3_PATH>/'
+  DATA_FORMAT     = 'PARQUET'
+  S3_ENDPOINT     = 's3.<REGION>.amazonaws.com'
+  CONNECTION_NAME = 'S3_CONNECTION'
+  PARALLELISM     = 'iproc()';
+```
+
+Like in import, you can increase the number of exporter processes. Since we need
+a dynamic number that the Exasol database can understand, you can combine the
+`iproc()` statement with `random()` and `floor()` operations.
+
+For example, to increase the exporter processes four times, set it as below:
+
+```sql
+PARALLELISM = 'iproc(), floor(random()*4)'
+```
+
+Please change this parameter according to your setup.
+
+Each exporter process creates a single file. This can be a problem if the table
+has many records. You can change this behavior by adapting the
+`EXPORT_BATCH_SIZE` parameter. This value is used to further split the number of
+records per process and create several files instead of a single file.
+
+### An Example Settings for Parallelism Parameter
+
+In this section, we are going to show you how to set an optional `PARALLELISM`
+parameter. In usual cases you do not have to set this parameter since it will be
+set automatically. However, occasionally it may be required that the users set
+this manually.
+
+Let's assume that we have these database resources, as shown in the picture
+below.
+
+![Cluster Resources Example](../images/cluster-resources.png)
+
+Each datanode has:
+- RAM: 376 GiB
+- Number of CPUs: 72
+
+With these known settings, we can set the parallelism parameter for import and
+export as follows.
+
+#### Import
+
+```
+PARALLELISM = 'nproc()*64'
+```
+
+#### Export
+
+```
+PARALLELISM = 'iproc(), floor(random()*64)'
+```
+
+This will set the maximum number of parallel processes to `64` and each process
+will have around `6 GiB (376 GiB / 64)` of RAM.
```

Comment:
I'm not an expert in memory model.
Are we sure that all RAM will be used by UDF?
There is SQL process heap, memory for data blocks, OS memory ...
How about a more flexible formulation here?

Reply:
How about?

```
This will set the maximum number of parallel processes to `64`. Additionally,
there is enough RAM `6 GiB (376 GiB / 64)` to use for importer/exporter and
other SQL processes.
```

Reply:
Maybe just
`and each process
will have up to 6 GiB (376 GiB / 64) of RAM.`
?

Reply:
Fixed